pkg/apps/alert/latest/alert_creater.go

```diff
@@ -60,11 +62,20 @@ func (ac *Creater) Ensure(alert *alertapi.Alert) error {
 	if err != nil {
 		return err
 	}
-	// Update components in cluster
-	commonConfig := crdupdater.NewCRUDComponents(ac.KubeConfig, ac.KubeClient, ac.Config.DryRun, false, alert.Spec.Namespace, cpList, "app=alert")
-	_, errors := commonConfig.CRUDComponents()
-	if len(errors) > 0 {
-		return fmt.Errorf("unable to update Alert components due to %+v", errors)
+	if strings.EqualFold(alert.Spec.DesiredState, "STOP") {
+		commonConfig := crdupdater.NewCRUDComponents(ac.KubeConfig, ac.KubeClient, ac.Config.DryRun, false, alert.Spec.Namespace,
+			&api.ComponentList{PersistentVolumeClaims: cpList.PersistentVolumeClaims}, "app=alert")
```

Comment:
This is mimicking blackduck stop for now.  @jdartigalongue @msenmurugan and I discussed that this should be made same as opssight for both blackduck and alert, in that when desired state is stop, scaled down everything to 0 as well as don't delete the PVC.  What do you think?

Reply:
I personally prefer deleting the resources when the instance is stopped but I don't have strong feelings about this. What are the advantages of scaling down everything to 0 over the way it currently works?

Reply:
Keeping as is for now.